### PR TITLE
MNT: Bump to v0.2.

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hhslepicka** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/Synchrotron-Radiation-Workshop/SRW/archive/v{{ version }}.tar.gz
-  sha256: 53bdcb0c02c59659bd0cb3849eac15c166db65b69d6fb62d4b8bf97cc4c96be8
+  sha256: 5b4d6969deb5d3965b9dcad50ba509ef4ed0cf342b4380eb25760705b1f3dde3
```
Comment (mrakitin) [inline recipe/meta.yaml:10]: Can we move the sha256 value to the set block on top and use the templated string here? I normally do it for the recipes I am maintaining, so it's easier to have the change of version and hash in one chunk.

**Review by mrakitin — COMMENTED** [review]
Thanks, @hhslepicka! I have a little improvement suggestion but looks good otherwise. Thanks for taking care of the tags.

File: recipe/meta.yaml (hunk 1 above)
Comment (hhslepicka) [reply recipe/meta.yaml]: Thanks @mrakitin for the review. Addressed via https://github.com/conda-forge/srwpy-feedstock/pull/5/commits/ea7b49bbf4903e162682a2da4f769b3491ee8f1a

**Review by mrakitin — APPROVED** [review]

**Pull request merged**